R/slide.R

```diff
@@ -309,15 +309,16 @@ epi_slide <- function(
     ...,
     .keep = TRUE
   ) %>%
-    bind_rows() %>%
-    filter(.real) %>%
-    select(-.real) %>%
+    {
+      vec_rbind(!!!.)
+    } %>%
+    `[`(.$.real, names(.) != ".real") %>%
```

Comment:
Is this another instance of avoiding tidyselect for performance?

Reply:
Yep.  It's also probably a bit more efficient just from combining row & col slicing when you have only a few columns.